modules/nw-kube-proxy-configuring.adoc

```diff
@@ -43,12 +43,45 @@ The syntax is validated by the `oc` command when you save the file and exit the
 editor. If your modifications contain a syntax error, the editor opens the file
 and displays an error message.
 
-. Run the following command to confirm that the configuration is successful:
+. Run the following command to confirm the configuration update:
++
+----
+$ oc get networks.operator.openshift.io -o yaml
+----
++
+The command will return output similar to the following example:
++
+[source,yaml]
+----
+apiVersion: v1
+items:
+- apiVersion: operator.openshift.io/v1
+  kind: Network
+  metadata:
+    name: cluster
+  spec:
+    clusterNetwork:
+    - cidr: 10.128.0.0/14
+      hostPrefix: 23
+    defaultNetwork:
+      type: OpenShiftSDN
+    deployKubeProxy: false
+    kubeProxyConfig:
+      bindAddress: 0.0.0.0
```

Comment:
deployKubeProxy: false
bindAddress:0.0.0.0 
those two are also need to remove.

Reply:
Removed.